admin_manual/release_schedule.rst

```diff
@@ -19,7 +19,7 @@ Maintenance releases are scheduled in a 4 week cycle with one week before the re
 Critical changes
 ----------------
 
-* PHP 7.4 is not supported anymore. Please upgrade to PHP 8.0 or higher.
+* PHP 8.0 is now deprecated. Please upgrade to PHP 8.1 or higher.
```

Comment:
I wonder if this is enough documentation for the deprecation and or if this is the only worthy of nothing this but I guessed this would be a good place to start!

Reply:
I would put it here as well:
https://docs.nextcloud.com/server/latest/admin_manual/installation/system_requirements.html
Add a note like `**(deprecated)**` on the PHP 8.0 one

Reply:
Right! Thanks!